src/resources/js/components/layout/SideBar.vue

```diff
@@ -1,82 +1,36 @@
 <template>
```

Comment:
It should be a bit wider. Plus according to the models, there isn't a sliding animation (which is very very annoying).